src/Sulu/Bundle/AdminBundle/Resources/views/CsvExport/form.html.twig

```diff
@@ -75,4 +75,7 @@
             </div>
         </div>
     </div>
+
+    {% block additions %}
```

Comment:
Do you think `additions` is a good name? Couldn't come up with something better... /cc @alexander-schranz @chirimoya @wachterjohannes